Classes/Issues/IssuesViewController.swift

```diff
@@ -286,6 +288,13 @@ final class IssuesViewController: MessageViewController,
         activityController.popoverPresentationController?.barButtonItem = sender
         present(activityController, animated: trueUnlessReduceMotionEnabled)
     }
+    
+    var insetWidth: CGFloat {
+        // assumptions here, but the collectionview may not have been laid out or content size found
+        // assume the collectionview is pinned to the view's bounds
+        let contentInset = feed.collectionView.contentInset
+        return view.bounds.width - contentInset.left - contentInset.right
+    }
```

Comment:
Moved this so that I could re-use it in the completion block after the fetch; see below 

```swift 
         if let current = strongSelf.result {
                    let issueResult = IssuesViewController.updateIssueResultModelTitle(
                        newTitle: newTitle,
                        oldTitle: strongSelf.currentIssueTitle ?? "",
                        username: strongSelf.client.userSession?.username ?? Constants.Strings.unknown,
                        issueResultModel: current,
                        width: strongSelf.insetWidth
                    )
                    strongSelf.client.cache.set(value: issueResult)
                }
```